.travis.yml

```diff
@@ -47,13 +45,13 @@ before_install:
   
   # Set up q for testing and execute tests on multiple 
   - if [[ $TESTS == "True" && "x$OD" != "x" && "x$QLIC_KC" != "x" ]]; then
-      export LD_LIBRARY_PATH=$BUILD_HOME/lib:$LD_LIBRARY_PATH;
-      export DYLD_LIBRARY_PATH=$BUILD_HOME/lib:$LD_LIBRARY_PATH;
-      export PATH=$BUILD_HOME/lib:$PATH;
-      curl -o qhome/q.zip -L $OD;
-      unzip -d qhome qhome/q.zip;
-      rm qhome/q.zip;
-      echo -n $QLIC_KC |base64 --decode > qhome/kc.lic;
+      export LD_LIBRARY_PATH=${MQTT_INSTALL_DIR}/lib:$LD_LIBRARY_PATH;
+      export DYLD_LIBRARY_PATH=${MQTT_INSTALL_DIR}/lib:$LD_LIBRARY_PATH;
+      export PATH=${MQTT_INSTALL_DIR}/lib:$PATH;
+      curl -o ${QHOME}/q.zip -L $OD;
+      unzip -d ${QHOME} ${QHOME}/q.zip;
+      rm ${QHOME}/q.zip;
+      echo -n $QLIC_KC |base64 --decode > ${QHOME}/kc.lic;
     else
```

Comment:
I think the old code was trying to a tmp q installation for testing.
If this is using the users QHOME it may overwrite various things that installed there which they may lose forever.

Reply:
This is not users QHOME.

```
- export QHOME=$(pwd)/qhome;
```
is QHOME set on travis container.